fix GEM geometry - HCAL emap - HCAL resoCorr HEP17 - SiStrip bad channels

franzoni opened the pull request:
This PR is incremental with respect to this other one https://github.com/cms-sw/cmssw/pull/17649 ; differences are updated [further down in this  conversation](https://github.com/cms-sw/cmssw/pull/17886#issuecomment-286271239) (since more commits have been added);
there the differences are described with respect to such PR.

arunhep committed:
GT updates for 900pre5

arunhep committed:
GTs updated

arunhep committed:
Run-2 MC Startup reverted back to 50ns JECs

Commit:
Merged refs/pull/17649/head from repository cms-sw

Commit:
fix GEM geometry - HCAL emap - HCAL resoCorr HEP17 - SiStrip bad channels

cmsbuild commented:
A new Pull Request was created by  @franzoni (Giovanni Franzoni) for CMSSW_9_0_X.

It involves the following packages:

Configuration/AlCa


@ghellwig, @arunhep, @cerminar, @cmsbuild, @franzoni, @mmusich, @davidlange6 can you please review it and eventually sign? Thanks.
@makortel, @Martin-Grunewald, @ghellwig, @tocheng this is something you requested to watch as well.
@davidlange6, @smuzaffar you are the release manager for this.

cms-bot commands are listed here cms-sw/cmssw#13028

arunhep commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: f6f60c3e6c341878bca0ebd208a36a4c442a68ce
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18342/summary.html

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/553f7beba0306ae7b7c385ed5b82452d65c7aac9
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18342/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18342/git-merge-result

cmsbuild commented:
Comparison job queued.

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18342/summary.html

franzoni commented:
**phaseI MC 2017:**
   * the change in geometry is expected to scatter differences widely, due to its impact on the pseudorandom number sequences
   * low level Sistrip plots hightly segmente -> can't see effect of the channel status changes over the 10 available events (pluse the previous bullet)
   * HCAL vs eta hit occupancies are the same or compatible (the emap change is expected to have no effect on digi/rehits)

**phaseI MC 2018:**
   * same picture as 2017

**phaseII MC:**
   * the change in channel status is not picked up by the Phase2OuterTracker plots . Tracking is also totally unaffected ( what impact to we expect ? @mmusic @dimattia )

**2016B**
   * differences in JEtMEt, Btag Physics ParticleFlow Btag Physics ParticleFlow -> as expected given the no extra change with respect to https://github.com/cms-sw/cmssw/pull/17649

mmusich commented:
@franzoni, my gitHub tag is @mmusich...
Coming to your question, I believe we expect (and see) no difference from the bad components change in 2023 workflows. I would be surprised of the contrary, given the geometry is completely different. I am expecting that at this level of development, OT conditions are hardwired. @boudoul to comment more. I suggested those to be included in the GT queue, as we already discussed verbally, *for your own convenience*, to ease the comparison with 2017 MC.

boudoul commented:
I'm not expecting changes in phase2 ,  or better :  I *do not want to see changes*  in phase2

franzoni commented:
Thanks @mmusich @boudul for the inputs

franzoni commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_9_0_X IBs (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @smuzaffar

franzoni commented:
@davidlange6 @smuzzafar
Having an IB including this request 
Will be very helpful to integrate quickly the last tags we received over the weekend
( Needed for the dog/Pog MC production campaign )



Thanks!

davidlange6 commented:
Hi @franzoni - sorry for the delay, focused on other releases lately. I see two items to comment on
 - it looks like these changes are not in 91x. They need to be there first
 - I believe it was decided not to push in the geometry changes into 90x. Did that decision get changed at some point? @ianna

ianna commented:
@davidlange6 - there were no GT queues for 91x yet when I submitted a request (see the e-mail thread of 4 days ago)

franzoni commented:
hello @david 

> it looks like these changes are not in 91x. They need to be there first
I can place the PR also in 91x , will do.

> decided not to push in the geometry changes into 90x
I interpreted the sentence "Seems fine but let's back to freezing the geometry" differently.
the two messages https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html followed shortly after , which we took as go ahead.

Given the nature of the change (GEM passive material)
AlCa/db was not particularly enthusiastic in putting them in, yet now that they are, we would prefer moving along w/o having to undo.

franzoni commented:
@davidlange6 

is the auto-forward port no longer in place ?

davidlange6 commented:
note that @david will likely reach some other user in github (who is now getting these mails.. sorry)

Ok, thanks - it sounds like @ianna needs more guidance about how to handle GT requests at release change overs.I'm sure it won't be the last time we are changing geometry in a pre1 without wanting the same change in the production release.

> On Mar 13, 2017, at 10:15 AM, Giovanni Franzoni <notifications@github.com> wrote:
> 
> hello @david
> 
> it looks like these changes are not in 91x. They need to be there first
> I can place the PR also in 91x , will do.
> 
> decided not to push in the geometry changes into 90x
> I interpreted the sentence "Seems fine but let's back to freezing the geometry" differently.
> the two messages https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html followed shortly after , which we took as go ahead.
> 
> Given the nature of the change (GEM passive material)
> AlCa/db was not particularly enthusiastic in putting them in, yet now that they are, we would prefer moving along w/o having to undo.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

davidlange6 commented:
no

> On Mar 13, 2017, at 10:20 AM, Giovanni Franzoni <notifications@github.com> wrote:
> 
> @davidlange6
> 
> is the auto-forward port no longer in place ?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

ianna commented:
@davidlange6 - yes, please

franzoni commented:
hello @davidlange6 @ianna 
we'll provide @ianna w/ guidance, though I believe she did notice that there were only 90x queues, which is already the correct logic.

@davidlange6 do you demand a revert of the GEM geometry or can we move along with what's in this PR ? 

> auto-forward port no longer in place ?
Pardon my ignorance: do I expect than the PR to be automatically placed (on what time scale) or should I take action ? 
We'll receive in the next hours tags from ECAL to be integrated (needed for the DPG/POG campaign), it would be great if we knew how/where to deploy them.

davidlange6 commented:
> On Mar 13, 2017, at 10:39 AM, Giovanni Franzoni <notifications@github.com> wrote:
> 
> hello @davidlange6 @ianna
> we'll provide @ianna w/ guidance, though I believe she did notice that there were only 90x queues, which is already the correct logic.
> 
> @davidlange6 do you demand a revert of the GEM geometry or can we move along with what's in this PR ?
> 

no- its ok for me - its a minor change.

> auto-forward port no longer in place ?
> Pardon my ignorance: do I expect than the PR to be automatically placed (on what time scale) or should I take action ?
> We'll receive in the next hours tags from ECAL to be integrated (needed for the DPG/POG campaign), it would be great if we knew how/where to deploy them.

You should take action. Just make a PR for the master branch.

> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

franzoni commented:
thanks @davidlange6 
we'll do.

arunhep committed:
data GTs updated with fixed GEM reco tag

arunhep committed:
MC Realistic GTs updated

arunhep committed:
Cosmic GT for 2018 updated

arunhep committed:
Cosmic GT updated

cmsbuild commented:
Pull request #17886 was updated. @ghellwig, @arunhep, @cerminar, @cmsbuild, @franzoni, @mmusich, @davidlange6 can you please check and sign again.

arunhep commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

franzoni commented:
This PR is incremental with respect to this other one https://github.com/cms-sw/cmssw/pull/17649 ; differences are described with respect to such PR. This description includes all the commits up to this point.


# Summary of changes in Global Tags

## Data

   * **runI and runII data** : [90X_dataRun2_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v6) as [90X_dataRun2_relval_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_relval_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_relval_v6/90X_dataRun2_relval_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII data relval** : [90X_dataRun2_relval_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_relval_v6) as [90X_dataRun2_PromptLike_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_PromptLike_v6/90X_dataRun2_PromptLike_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII data prompt-like** : [90X_dataRun2_PromptLike_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v6) as [90X_dataRun2_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_v6/90X_dataRun2_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runI and runII HLT** : [90X_dataRun2_HLT_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v4) as [90X_dataRun2_HLT_frozen_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_frozen_v4/90X_dataRun2_HLT_frozen_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runI and runII HLT relval** : [90X_dataRun2_HLT_relval_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v4) as [90X_dataRun2_HLT_relval_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_relval_v4/90X_dataRun2_HLT_relval_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII HLT HI** : [90X_dataRun2_HLTHI_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v4) as [90X_dataRun2_HLTHI_frozen_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLTHI_frozen_v4/90X_dataRun2_HLTHI_frozen_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note



## Upgrade

   * **PhaseII realistic scenario** : [90X_upgrade2023_realistic_v8](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2023_realistic_v8) as [90X_upgrade2023_realistic_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2023_realistic_v6) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2023_realistic_v8/90X_upgrade2023_realistic_v6):
      * removed unnecessary geometry records
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/

   * **PhaseI 2017 cosmics peak scenario** : [90X_upgrade2017cosmics_realistic_peak_v17](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v17) as [90X_upgrade2017cosmics_realistic_peak_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v15) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v17/90X_upgrade2017cosmics_realistic_peak_v15):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html


   * **PhaseI 2017 cosmics scenario** : [90X_upgrade2017cosmics_realistic_deco_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_deco_v16) as [90X_upgrade2017cosmics_realistic_peak_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v15) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017cosmics_realistic_deco_v16/90X_upgrade2017cosmics_realistic_peak_v15):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html


   * **PhaseI 2017 design scenario** : [90X_upgrade2017_design_IdealBS_v18](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v18) as [90X_upgrade2017_design_IdealBS_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v16) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_design_IdealBS_v18/90X_upgrade2017_design_IdealBS_v16):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html


   * **PhaseI 2017 realistic scenario** : [90X_upgrade2017_realistic_v18](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v18) as [90X_upgrade2017_realistic_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v16) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_realistic_v18/90X_upgrade2017_realistic_v16):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html


   * **PhaseI 2018 cosmics scenario** : [90X_upgrade2018cosmics_realistic_deco_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_deco_v15) as [90X_upgrade2018cosmics_realistic_peak_v12](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_peak_v12) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018cosmics_realistic_deco_v15/90X_upgrade2018cosmics_realistic_peak_v12):
      * updated geometry (fix GEM passive volume and HCAL changes from code) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * sistrip noise switched from peak to DECO (the new default)
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html

   * **PhaseI 2018 design scenario** : [90X_upgrade2018_design_IdealBS_v14](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v14) as [90X_upgrade2018_design_IdealBS_v13](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v13) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_design_IdealBS_v14/90X_upgrade2018_design_IdealBS_v13):
      * updated geometry (fix GEM passive volume and HCAL changes from code) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html


   * **PhaseI 2018 realistic scenario** : [90X_upgrade2018_realistic_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v15) as [90X_upgrade2018_realistic_v13](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v13) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_realistic_v15/90X_upgrade2018_realistic_v13):
      * updated geometry (fix GEM passive volume and HCAL changes from code) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html

cmsbuild commented:
+1
Tested at: 53fc7fd98c70a224213e0cf9f0b2334cdaed40a3
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18380/summary.html

cmsbuild commented:
Comparison job queued.

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18380/summary.html

franzoni commented:
**phaseI MC 2017:**
   * the change in geometry is expected to scatter differences widely, due to its impact on the pseudorandom number sequences
   * low level Sistrip plots hightly segmented -> can't see effect of the channel status changes over the 10 available events (plus the previous bullet)
   * HCAL vs eta hit occupancies are the same or compatible (the emap change is expected to have no effect on digi/rehits)
   * change in occupancy clear in EB, as a result of SelRead and Transparency/Pedestal changes

**phaseI MC 2018:**
   * same picture as 2017
   * change in occupancy clear in EE, as a result of SelRead and Transparency/Pedestal changes

**phaseII MC:**
   * the change in channel status is not picked up by the Phase2OuterTracker plots . Tracking is also totally unaffected --> expected

**data:**
   * differences in JEtMEt, Btag Physics ParticleFlow Btag Physics ParticleFlow -> as expected given the no extra change with respect to https://github.com/cms-sw/cmssw/pull/17649
   * 2012B and 2016B unaffected by replacing for GEM reco geometry the IOV 1-inf with 287446-inf

arunhep commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_9_0_X IBs (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @smuzaffar

arunhep committed:
ECAL SR seeting tag fix

cmsbuild commented:
Pull request #17886 was updated. @ghellwig, @arunhep, @cerminar, @cmsbuild, @franzoni, @mmusich, @davidlange6 can you please check and sign again.

franzoni commented:
This PR is incremental with respect to this other one https://github.com/cms-sw/cmssw/pull/17649 ; differences are described with respect to such PR. This description includes all the commits up to this point.


# Summary of changes in Global Tags

## Data

   * **runI and runII data** : [90X_dataRun2_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v6) as [90X_dataRun2_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_v6/90X_dataRun2_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII data relval** : [90X_dataRun2_relval_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_relval_v6) as [90X_dataRun2_relval_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_relval_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_relval_v6/90X_dataRun2_relval_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII data prompt-like** : [90X_dataRun2_PromptLike_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v6) as [90X_dataRun2_PromptLike_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_PromptLike_v6/90X_dataRun2_PromptLike_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runI and runII HLT** : [90X_dataRun2_HLT_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v4) as [90X_dataRun2_HLT_frozen_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_frozen_v4/90X_dataRun2_HLT_frozen_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runI and runII HLT relval** : [90X_dataRun2_HLT_relval_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v4) as [90X_dataRun2_HLT_relval_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_relval_v4/90X_dataRun2_HLT_relval_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII HLT HI** : [90X_dataRun2_HLTHI_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v4) as [90X_dataRun2_HLTHI_frozen_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLTHI_frozen_v4/90X_dataRun2_HLTHI_frozen_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note



## Upgrade

   * **PhaseII realistic scenario** : [90X_upgrade2023_realistic_v9](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2023_realistic_v9) as [90X_upgrade2023_realistic_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2023_realistic_v6) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2023_realistic_v9/90X_upgrade2023_realistic_v6):
      * removed unnecessary geometry records
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (fix), ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/

   * **PhaseI 2017 cosmics peak scenario** : [90X_upgrade2017cosmics_realistic_peak_v18](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v18) as [90X_upgrade2017cosmics_realistic_peak_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v15) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v18/90X_upgrade2017cosmics_realistic_peak_v15):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html


   * **PhaseI 2017 cosmics scenario** : [90X_upgrade2017cosmics_realistic_deco_v17](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_deco_v17) as [90X_upgrade2017cosmics_realistic_peak_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v15) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017cosmics_realistic_deco_v17/90X_upgrade2017cosmics_realistic_peak_v15):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html


   * **PhaseI 2017 design scenario** : [90X_upgrade2017_design_IdealBS_v18](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v18) as [90X_upgrade2017_design_IdealBS_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v16) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_design_IdealBS_v18/90X_upgrade2017_design_IdealBS_v16):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html


   * **PhaseI 2017 realistic scenario** : [90X_upgrade2017_realistic_v19](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v19) as [90X_upgrade2017_realistic_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v16) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_realistic_v19/90X_upgrade2017_realistic_v16):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html


   * **PhaseI 2018 cosmics scenario** : [90X_upgrade2018cosmics_realistic_deco_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_deco_v16) as [90X_upgrade2018cosmics_realistic_peak_v12](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_peak_v12) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018cosmics_realistic_deco_v16/90X_upgrade2018cosmics_realistic_peak_v12):
      * updated geometry (fix GEM passive volume and HCAL changes from code) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * sistrip noise switched from peak to DECO (the new default)
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html

   * **PhaseI 2018 design scenario** : [90X_upgrade2018_design_IdealBS_v14](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v14) as [90X_upgrade2018_design_IdealBS_v13](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v13) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_design_IdealBS_v14/90X_upgrade2018_design_IdealBS_v13):
      * updated geometry (fix GEM passive volume and HCAL changes from code) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html


   * **PhaseI 2018 realistic scenario** : [90X_upgrade2018_realistic_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v16) as [90X_upgrade2018_realistic_v13](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v13) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_realistic_v16/90X_upgrade2018_realistic_v13):
      * updated geometry (fix GEM passive volume and HCAL changes from code) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: 584b3453c9b6b66afe53e5cdd546867736997b32
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18424/summary.html

cmsbuild commented:
Comparison job queued.

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18424/summary.html

franzoni commented:
The doubt expressed earlier about missing changes in EE 
stems from the fact that the relmon differences are run with a loose bin-to-bin "pseudo-chi2" selection (see relmon doc https://twiki.cern.ch/twiki/bin/view/CMSPublic/RelMon#Statistical_Tests ), which hides changes when they're actually there .

Below the chance in occupancy in EB (already found in relmon) and EE (not).
This solves the remaining doubts.
We'll soon do the same cross check for 91x

[EEPlusOT_90X_RedTarget_BlueRef_digiOccupancy_projectionPhi.pdf](https://github.com/cms-sw/cmssw/files/845668/EEPlusOT_90X_RedTarget_BlueRef_digiOccupancy_projectionPhi.pdf)
[EBOT_90X_RedTarget_BlueRef_digiOccupancy_projectionPhi.pdf](https://github.com/cms-sw/cmssw/files/845667/EBOT_90X_RedTarget_BlueRef_digiOccupancy_projectionPhi.pdf)

franzoni commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_9_0_X IBs (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @smuzaffar

arunhep committed:
2017 and 2018 GTs updated with HCAL LUTs Bug fix and Resp Corr

cmsbuild commented:
Pull request #17886 was updated. @ghellwig, @arunhep, @cerminar, @cmsbuild, @franzoni, @mmusich, @davidlange6 can you please check and sign again.

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

franzoni commented:
This PR is incremental with respect to this other one https://github.com/cms-sw/cmssw/pull/17649 ; differences are described with respect to such PR. This description includes all the commits up to this point.


# Summary of changes in Global Tags

## Data

   * **runI and runII data** : [90X_dataRun2_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v6) as [90X_dataRun2_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_v6/90X_dataRun2_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII data relval** : [90X_dataRun2_relval_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_relval_v6) as [90X_dataRun2_relval_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_relval_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_relval_v6/90X_dataRun2_relval_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII data prompt-like** : [90X_dataRun2_PromptLike_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v6) as [90X_dataRun2_PromptLike_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_PromptLike_v6/90X_dataRun2_PromptLike_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runI and runII HLT** : [90X_dataRun2_HLT_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v4) as [90X_dataRun2_HLT_frozen_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_frozen_v4/90X_dataRun2_HLT_frozen_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runI and runII HLT relval** : [90X_dataRun2_HLT_relval_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v4) as [90X_dataRun2_HLT_relval_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_relval_v4/90X_dataRun2_HLT_relval_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII HLT HI** : [90X_dataRun2_HLTHI_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v4) as [90X_dataRun2_HLTHI_frozen_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLTHI_frozen_v4/90X_dataRun2_HLTHI_frozen_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note



## Upgrade

   * **PhaseII realistic scenario** : [90X_upgrade2023_realistic_v9](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2023_realistic_v9) as [90X_upgrade2023_realistic_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2023_realistic_v6) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2023_realistic_v9/90X_upgrade2023_realistic_v6):
      * removed unnecessary geometry records
      * [sistrip bad components](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html) 
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (fix), ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/

   * **PhaseI 2017 cosmics peak scenario** : [90X_upgrade2017cosmics_realistic_peak_v19](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v19) as [90X_upgrade2017cosmics_realistic_peak_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v15) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v19/90X_upgrade2017cosmics_realistic_peak_v15):
      * [updated geometry (fix GEM passive volume)](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html) 
      * [HcalElectronicsMap](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html) 
      * [HcalRespCorrs 1 -> 1.05](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html) 
      * Bug Fix in HCAL LUTs described and HCAL Response Correction Update [details here](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2860.html)
      * [sistrip bad components](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html) 
      * [beamstpot consistenet w/ alignement ](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html)
      * ECAL: [2017 predictions for](https://indico.cern.ch/event/615692/contributions/2484421/) channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - 
      * [Drift tubes alignment and alignment errors ](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html)


   * **PhaseI 2017 cosmics scenario** : [90X_upgrade2017cosmics_realistic_deco_v18](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_deco_v18) as [90X_upgrade2017cosmics_realistic_peak_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v15) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017cosmics_realistic_deco_v18/90X_upgrade2017cosmics_realistic_peak_v15):
      * [updated geometry](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html) (fix GEM passive volume) 
      * [HcalElectronicsMap](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html) 
      * [HcalRespCorrs 1 -> 1.05](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html) 
      * Bug Fix in HCAL LUTs described and HCAL Response Correction Update [details here](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2860.html)
      * [sistrip bad components](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html) 
      * [beamstpot consistenet w/ alignement](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html) 
      * ECAL: [2017 predictions](https://indico.cern.ch/event/615692/contributions/2484421/) for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - 
      * [Drift tubes alignment and alignment errors ](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html)


   * **PhaseI 2017 design scenario** : [90X_upgrade2017_design_IdealBS_v19](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v19) as [90X_upgrade2017_design_IdealBS_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v16) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_design_IdealBS_v19/90X_upgrade2017_design_IdealBS_v16):
      * [updated geometry](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html) (fix GEM passive volume) 
      * [HcalElectronicsMap](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html) 
      * [HcalRespCorrs 1 -> 1.05](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html) 
      * Bug Fix in HCAL LUTs described and HCAL Response Correction Update [details here](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2860.html)

   * **PhaseI 2017 realistic scenario** : [90X_upgrade2017_realistic_v20](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v20) as [90X_upgrade2017_realistic_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v16) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_realistic_v20/90X_upgrade2017_realistic_v16):
      * updated geometry (fix [GEM passive volume](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html)) 
      * [HcalElectronicsMap]( https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html)
      * [HcalRespCorrs 1 -> 1.05](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html) 
      * Bug Fix in HCAL LUTs described and HCAL Response Correction Update [details here](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2860.html)
      * [sistrip bad components](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html) 
      * [beamstpot consistenet w/ alignement](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html) 
      * ECAL: [2017 predictions](https://indico.cern.ch/event/615692/contributions/2484421/) for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - 
      * [Drift tubes alignment and alignment errors ](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html
)

   * **PhaseI 2018 cosmics scenario** : [90X_upgrade2018cosmics_realistic_deco_v17](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_deco_v17) as [90X_upgrade2018cosmics_realistic_peak_v12](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_peak_v12) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018cosmics_realistic_deco_v17/90X_upgrade2018cosmics_realistic_peak_v12):
      * [updated geometry](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html) (fix GEM passive volume and HCAL changes from code) 
      * [HcalElectronicsMap](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html) 
      * Bug Fix in HCAL LUTs described and HCAL Response Correction Update [details here](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2860.html)
      * [sistrip bad components ](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html)
      * sistrip noise switched from peak to DECO (the new default)
      * [beamstpot consistenet w/ alignement](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html) 
      * ECAL: [2017 predictions](https://indico.cern.ch/event/615692/contributions/2484421/) for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - 
      * [Drift tubes alignment and alignment errors ](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html)

   * **PhaseI 2018 design scenario** : [90X_upgrade2018_design_IdealBS_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v15) as [90X_upgrade2018_design_IdealBS_v13](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v13) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_design_IdealBS_v15/90X_upgrade2018_design_IdealBS_v13):
      * [updated geometry](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html) (fix GEM passive volume and HCAL changes from code) 
      * [HcalElectronicsMap](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html) 
      * Bug Fix in HCAL LUTs described and HCAL Response Correction Update [details here](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2860.html)

   * **PhaseI 2018 realistic scenario** : [90X_upgrade2018_realistic_v17](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v17) as [90X_upgrade2018_realistic_v13](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v13) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_realistic_v17/90X_upgrade2018_realistic_v13):
      * [updated geometry](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html) (fix GEM passive volume and HCAL changes from code) 
      * [HcalElectronicsMap](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html) 
      * Bug Fix in HCAL LUTs described and HCAL Response Correction Update [details here](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2860.html)
      * [sistrip bad components](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html)
      * [beamstpot consistenet w/ alignement](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html) 
      * ECAL: [2017 predictions](https://indico.cern.ch/event/615692/contributions/2484421/) for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout (*fix*), ES calibration - 
      * [Drift tubes alignment and alignment errors](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html)

cmsbuild commented:
+1
Tested at: 074e25f271d16a6716980284e2de2434b8ccd264
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18536/summary.html

cmsbuild commented:
Comparison job queued.

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17886/18536/summary.html

franzoni commented:
https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2860/1/1/1/2/1/1/1/1.html

franzoni commented:
Latest update related to hcal trigger primitives discussed here https://github.com/cms-sw/cmssw/pull/17975#issuecomment-287603778

franzoni commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_9_0_X IBs (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @smuzaffar

davidlange6 commented:
+1

franzoni commented:
hello @davidlange6 @ianna 
we take note of https://github.com/cms-sw/cmssw/pull/17978#issuecomment-287713193 
cheers, G.
